Include/cpython/unicodeobject.h

```diff
@@ -1214,13 +1214,18 @@ Py_DEPRECATED(3.3) PyAPI_FUNC(Py_UNICODE*) PyUnicode_AsUnicodeCopy(
     );
 
 /* Return an interned Unicode object for an Identifier; may fail if there is no memory.*/
+#if !defined(Py_LIMITED_API)
 PyAPI_FUNC(PyObject*) _PyUnicode_FromId(_Py_Identifier*);
+#endif
 /* Clear all static strings. */
+#if !defined(Py_LIMITED_API)
 PyAPI_FUNC(void) _PyUnicode_ClearStaticStrings(void);
```

Comment:
I don't think that we should expose this function. It's only called by _PyUnicode_Fini(). I don't think that it's useful to clear _PyUnicode_FromId() cache. Some C extensions even rely on the fact that _PyUnicode_FromId() cannot fail after _PyUnicode_FromId() has been called the first time. I suggest to make the function static in unicodeobject.c.